stock_demand_estimate/wizards/stock_demand_estimate_wizard_view.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0"?>
+<odoo>
+
+        <record model="ir.ui.view"
+                id="view_stock_demand_estimate_sheet_form">
+            <field name="name">stock.demand.estimate.sheet.form</field>
+            <field name="model">stock.demand.estimate.sheet</field>
+            <field name="arch" type="xml">
+                <form string="Stock Demand Estimate Sheet">
+                    <group>
+                        <group name="dates">
+                            <label for="date_start" string="Timesheet Period"/>
+                            <div><field name="date_start" class="oe_inline"/> to <field name="date_end" class="oe_inline"/></div>
+                        </group>
+                        <newline/>
+                        <group name="attributes">
+                            <field name="date_range_type_id"/>
+                            <field name="location_id"/>
+                        </group>
+                    </group>
+                    <div/>
+                    <group name="estimated_quantity"
+                           string="Estimated quantity">
+                        <field name="line_ids" nolabel="1"
+                               widget="x2many_2d_matrix"
+                               field_x_axis="value_x"
+                               field_y_axis="value_y"
+                               field_value="product_uom_qty"
+                               x_axis_clickable="0"
+                               y_axis_clickable="0">
+                            <tree>
+                                <field name="value_x"/>
+                                <field name="value_y"/>
+                                <field name="product_uom_qty"/>
+                            </tree>
+                        </field>
+                    </group>
+                    <footer>
+                    <button name="button_validate"
+                            type="object"
+                            string="Validate"
+                            class="oe_highlight"/>
+                    <button class="oe_link"
+                            special="cancel"
+                            string="Cancel"/>
+                    </footer>
+                </form>
+          </field>
+        </record>
+
+        <record model="ir.ui.view"
+                id="view_demand_estimate_wizard_form">
+            <field name="name">stock.demand.estimate.wizard.form</field>
+            <field name="model">stock.demand.estimate.wizard</field>
+            <field name="arch" type="xml">
+                <form string="Stock Demand Estimate Wizard">
+                    <group>
+                        <group name="dates">
+                            <label for="date_start" string="Period"/>
+                            <div><field name="date_start" class="oe_inline"/> to <field name="date_end" class="oe_inline"/></div>
```

Comment:
These fields don't appear on the same line.

Reply:
well, i am not sure they should :D

Reply:
`class="oe_inline"` attr is there for that. :wink: 
The `<group>` creates a two column layout, so it should look much better if you remove the empty one on line 57.
However, you might have to add a <group> for the other fields.